### PR TITLE
pw-link: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
I could not find dedicated documentation of this tool anywhere so I quickly whipped up this page for the few use cases I started using it for. Even the "More information" page doesn't really have anything. The only place that has actual info was the nixos wiki https://nixos.wiki/wiki/PipeWire

**Comment by Managor** [comment]
And do take into account that the tool regards mics as outputs and speakers as inputs as it's from the perspective of the pipewire system.

File: pages/linux/pw-link.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pw-link
+
+> Tool for managing links between ports in pipewire.
+> More information: <https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Virtual-Devices>.
+
+- Display the help page:
```
Comment (CleanMachine1) [inline pages/linux/pw-link.md:6]:
```suggestion
- Display help:
```

File: pages/linux/pw-link.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# pw-link
+
+> Tool for managing links between ports in pipewire.
```
Comment (blueskyson) [inline pages/linux/pw-link.md:3]: Use precise form of technical terms.
```suggestion
> Manage links between ports in PipeWire.
```

**Review by blueskyson — COMMENTED** [review]
Please sign the [contribution license](https://cla-assistant.io/tldr-pages/tldr) to proceed pull request.

File: pages/linux/pw-link.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display help:
+
+`pw-link -h`
+
+- List all audio output and input ports:
+
+`pw-link --output --input'`
+
+- Create a link between an output and an input ports:
```
Comment (CleanMachine1) [inline pages/linux/pw-link.md:14]:
```suggestion
- Create a link between an output and an input port:
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**